### PR TITLE
[docs] remove the installation linking step from the GitHub builds guide and optimize images

### PR DESCRIPTION
**Pull request by FiberJW**
# Why

An update to the GitHub linking flow removed the installation linking step.

# How

- I removed the part of the GitHub builds guide detailing the installation linking step.
- I added a section for adding more GitHub accounts to expand your repo selection candidates
- I split up a note detailing the requirements for linking to more relevant points in the flow
- I ran `ImageOptim` on all of the images in the GitHub builds directory

# Test Plan

http://localhost:3002/build/building-from-github/#configure-your-app-for-github

Screenshot 2024-03-18 at 17 38 55

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/build/building-from-github.mdx [hunk 1]
```diff
@@ -59,25 +59,28 @@ page](https://expo.dev/accounts/[account]/projects/[projectName]/github).
 
 Install the Expo GitHub App on your GitHub account.
 
+> **Note:** You must be an admin of the Expo account to install the app.
```
Comment (amandeepmittal) [inline docs/pages/build/building-from-github.mdx:62]:
```suggestion
> **Note:** You must have [Owner or Admin access](/accounts/working-together/#manage-access) of the Expo account to install the app.
```

Let's link to Owner and Admin role and also specify Owner and Admin both so as not leave any room for confusion.

File: docs/pages/build/building-from-github.mdx [hunk 2]
```diff
@@ -59,25 +59,28 @@ page](https://expo.dev/accounts/[account]/projects/[projectName]/github).
...
+
 <ImageSpotlight
   alt="The GitHub app installation UI"
   src="/static/images/eas-build/build-from-github/install-github-app.png"
 />
 
-Link your GitHub account to your Expo account.
+Then, you'll be able to link your repository to your Expo project.
```
Comment (amandeepmittal) [inline docs/pages/build/building-from-github.mdx:69]:
```suggestion
Then, link the GitHub repository to your Expo project.
```

File: docs/pages/build/building-from-github.mdx [hunk 3]
```diff
@@ -59,25 +59,28 @@ page](https://expo.dev/accounts/[account]/projects/[projectName]/github).
...
 
-> **Note:** You must be an admin of the Expo account to install the app. Also, you can only link GitHub organizations to Expo organizations.
+> **Note:** You can only link GitHub organization repositoriess to Expo organizations.
```
Comment (amandeepmittal) [inline docs/pages/build/building-from-github.mdx:71]:
```suggestion
> **Note:** You can only link [GitHub organization repositories](https://docs.github.com/en/organizations) to Expo organizations.
```

- Add a link to GitHub organization repo's
- Fix typo

File: docs/pages/build/building-from-github.mdx [hunk 4]
```diff
@@ -59,25 +59,28 @@ page](https://expo.dev/accounts/[account]/projects/[projectName]/github).
...
 
 <ImageSpotlight
-  alt="The GitHub account linking UI"
-  src="/static/images/eas-build/build-from-github/link-github-account.png"
+  alt="The repository selector on the Expo project GitHub settings page"
+  src="/static/images/eas-build/build-from-github/connect-a-repository.png"
 />
 
-Then, you'll be able to link your repository to your Expo project.
+If you want to add a repository from a different GitHub account, click the "Add new account" option
```
Comment (amandeepmittal) [inline docs/pages/build/building-from-github.mdx:78]:
```suggestion
To add a repository from a different GitHub account, click the **Add new account** option in the account selector dropdown.
```

- Let's be concise here
- expo.dev settings are bold in our docs

File: docs/pages/build/building-from-github.mdx [hunk 5]
```diff
@@ -59,25 +59,28 @@ page](https://expo.dev/accounts/[account]/projects/[projectName]/github).
...
+in the account selector dropdown.
```
Comment (amandeepmittal) [inline docs/pages/build/building-from-github.mdx:79]:
```suggestion
```

Added this in the above LOC by applying Prettier formatting.

File: docs/pages/build/building-from-github.mdx [hunk 6]
```diff
@@ -59,25 +59,28 @@ page](https://expo.dev/accounts/[account]/projects/[projectName]/github).
...
+Then, you'll be able to link your repository to your Expo project.
 
-> **Note:** You must be an admin of the Expo account to install the app. Also, you can only link GitHub organizations to Expo organizations.
+> **Note:** You can only link GitHub organization repositoriess to Expo organizations.
 
 <ImageSpotlight
-  alt="The GitHub account linking UI"
-  src="/static/images/eas-build/build-from-github/link-github-account.png"
+  alt="The repository selector on the Expo project GitHub settings page"
+  src="/static/images/eas-build/build-from-github/connect-a-repository.png"
```
Comment (amandeepmittal) [inline docs/pages/build/building-from-github.mdx:75]:
```suggestion
  src="/static/images/eas-build/build-from-github/connect-a-repository.png"
  style={{ width: 600 }}
```
The image rendered here is a bit large. Let's add a width.

**Review by Simek — APPROVED** [review]

File: docs/pages/build/building-from-github.mdx [hunk 7]
```diff
@@ -59,25 +59,28 @@ page](https://expo.dev/accounts/[account]/projects/[projectName]/github).
...
 
-Then, you'll be able to link your repository to your Expo project.
+If you want to add a repository from a different GitHub account, click the "Add new account" option
+in the account selector dropdown.
 
 <ImageSpotlight
-  alt="The repository selector on the Expo project GitHub settings page"
-  src="/static/images/eas-build/build-from-github/connect-a-repository.png"
+  alt="The account selector on the Expo project GitHub settings page"
+  src="/static/images/eas-build/build-from-github/add-new-account.png"
```
Comment (amandeepmittal) [inline docs/pages/build/building-from-github.mdx:83]:
```suggestion
  src="/static/images/eas-build/build-from-github/add-new-account.png"
  style={{ width: 400 }}
```

Same as above.

**Review by amandeepmittal — APPROVED** [review]

**Pull request merged**